packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -223,7 +227,10 @@ class _GoogleMapState extends State<GoogleMap> {
       'polygonsToAdd': serializePolygonSet(widget.polygons),
       'polylinesToAdd': serializePolylineSet(widget.polylines),
       'circlesToAdd': serializeCircleSet(widget.circles),
+      'creationMapId': _creationMapId,
```

Comment:
Can web just use the platform view id instead?

Reply:
These are the parameters that are passed in to create the view; look a little bit below to the call: `_googleMapsFlutterPlatform.buildView...`, that's where the view ID would be eventually created.

I tried to keep this fix self-contained to the web plugin, but I couldn't find how. I need a constant cache key for the widget and since the web view was being "recreated" all the time, I couldn't find *where* to get this value from (tried to get `key` from the widget or the parent, but if not passed those are "null").

I couldn't find a better fix than this one, given how the Map widgets are created  :/